install

```diff
@@ -77,6 +77,15 @@ def macos_version
   @macos_version ||= Version.new(`/usr/bin/sw_vers -productVersion`.chomp[/10\.\d+/])
 end
 
+def should_install_command_line_tools
+  if macos_version >= "10.9"
+    developer_dir = `/usr/bin/xcode-select -print-path 2>/dev/null`.chomp
+    return developer_dir.empty? || !File.exist?("#{developer_dir}/usr/bin/git")
+  else
+    return false
```

Comment:
Don't need the `return` here; it's implicit in Ruby. Also might be nicer to just do `return false if macos_version < "10.9"` above?